### PR TITLE
Refactoring - Sorting imports with isort and linting with black

### PR DESCRIPTION
**Pull request by paulosgf**
New adjustment.

## Description
Refactoring - Sorting imports with isort and linting with black

## How Has This Been Tested?
Tested on local fork with several plugins

## Additional context

Add any other context or screenshots about the pull request here.

## Checklist

- [ x] I have read, understood and followed our [Contributing Guide](https://github.com/pyhoneybot/honeybot/blob/master/CONTRIBUTING.md)
- [ x] My code follows the code style of this project
- [ x] I have updated the documentation accordingly
- [ x] I added my country flag to the `README.md`

## Code of Conduct

By submitting this issue, you agree to follow our [Code of Conduct](https://github.com/pyhoneybot/honeybot/blob/master/CODE_OF_CONDUCT.md)

- [x] I agree to follow this project's Code of Conduct

### CODE REVIEW

File: src/honeybot/plugins/downloaded/fact/main.py [hunk 1]
```diff
@@ -39,8 +39,7 @@ def run(self, incoming, methods, info, bot_info):
     + "Instead, they played piano music."
     "Amelia Earhart and Eleanor Roosevelt once sneaked out of a White"
     + "House event, commandeered an airplane, and went on a joyride to Baltimore."
-    "The 100 folds in a chef's toque are said to represent 100 ways "
```
Comment (MrTanoshii) [inline src/honeybot/plugins/downloaded/fact/main.py:42]: There's no need to use `+` if it is a single string

File: src/honeybot/plugins/downloaded/flight/main.py [hunk 2]
```diff
@@ -38,8 +38,7 @@ def run(self, incoming, methods, info, bot_info):
                     if "errors" in flight:  # check for api error
                         methods["send"](
                             info["address"],
-                            "Invalid input! Callsign "
-                            + "should not be more than 10 characters!",
+                            "Invalid input! Callsign " + "should not be more than 10 characters!",
```
Comment (MrTanoshii) [inline src/honeybot/plugins/downloaded/flight/main.py:41]: There's no need to use + if it is a single string

File: src/honeybot/plugins/downloaded/joke/main.py [hunk 3]
```diff
@@ -42,16 +42,13 @@ def joke(self):
             "What do you call a fake noodle? An Impasta.",
             "Did you hear about the restaurant on the moon?"
             + " It had great food but no atmosphere.",
-            "Why do the put fences around grave yards?"
-            + " Because people are dying to get in.",
+            "Why do the put fences around grave yards?" + " Because people are dying to get in.",
```
Comment (MrTanoshii) [inline src/honeybot/plugins/downloaded/joke/main.py:45]: There's no need to use + if it is a single string

File: src/honeybot/plugins/downloaded/mail/main.py [hunk 4]
```diff
@@ -105,9 +105,7 @@ def run(self, incoming, methods, info, bot_info):
                     SUBJECT_INDEX = msgs.index(
                         ".subject"
                     )  # gets index for ".subject" used in body, and subject functions
-                    MAX_INDEX = len(
-                        msgs
-                    )  # gets max index to know the max indexing value
+                    MAX_INDEX = len(msgs)  # gets max index to know the max indexing value
```
Comment (MrTanoshii) [inline src/honeybot/plugins/downloaded/mail/main.py:108]: Comment is redundant

File: src/honeybot/plugins/downloaded/quote/main.py [hunk 5]
```diff
@@ -45,14 +45,12 @@ def quote(self):
             "He that falls in love with himself will have no rivals.",
             "Nothing is impossible, the word itself says 'I'm possible'!",
             "It is hard to fail, but it is worse never to have tried to succeed.",
-            "Don't judge each day by the harvest you reap "
-            + "but by the seeds that you plant.",
+            "Don't judge each day by the harvest you reap " + "but by the seeds that you plant.",
```
Comment (MrTanoshii) [inline src/honeybot/plugins/downloaded/quote/main.py:48]: There's no need to use + if it is a single string

File: src/honeybot/plugins/downloaded/riddle/main.py [hunk 6]
```diff
@@ -64,13 +62,11 @@ def __init__(self):
             "What has a head, a tail, is brown, and has no legs?": "a penny",
             "What English word has three consecutive double letters?": "ookkeeper",
             "I know a word of letters three. Add two, and fewer there will be!": "few",
-            "Two in a corner, 1 in a room, 0 in a house,"
-            + " but 1 in a shelter. What am I?": "r",
+            "Two in a corner, 1 in a room, 0 in a house," + " but 1 in a shelter. What am I?": "r",
```
Comment (MrTanoshii) [inline src/honeybot/plugins/downloaded/riddle/main.py:65]: There's no need to use + if it is a single string

File: src/honeybot/plugins/downloaded/selfTrivia/main.py [hunk 7]
```diff
@@ -23,13 +23,10 @@ def __init__(self):
 
     def trivia(self):
         questions = [
-            "Venetian painter Jacopo Robusti was "
-            + "better known by what name?",  # Tintoretto
-            "In what town was FDR's "
-            + '"Little White House" located?',  # Warm Springs, Georgia
+            "Venetian painter Jacopo Robusti was " + "better known by what name?",  # Tintoretto
```
Comment (MrTanoshii) [inline src/honeybot/plugins/downloaded/selfTrivia/main.py:26]: There's no need to use + if it is a single string

**Review by MrTanoshii — COMMENTED** [review]
Is the move from "flake8" to "black" without retaining the properties of https://github.com/pyhoneybot/honeybot/blob/master/setup.cfg intentional?

**Comment by paulosgf** [comment]
> Is the move from "flake8" to "black" without retaining the properties of https://github.com/pyhoneybot/honeybot/blob/master/setup.cfg intentional?

@MrTanoshii, I based on the project's original configs made by @Abdur-rahmaanJ, here: https://github.com/pyhoneybot/honeybot/blob/master/.pep8speaks.yml
There is a 100 char per line length limit.

File: src/honeybot/plugins/downloaded/fact/main.py (hunk 1 above)
Comment (paulosgf) [reply src/honeybot/plugins/downloaded/fact/main.py:42]: > There's no need to use `+` if it is a single string

These "+" was to obey the 100 char limit. I guess that new lint pass modified the original flake8 style

**Pull request merged**